server/odc-service/src/main/java/com/oceanbase/odc/service/iam/auth/CustomAuthenticationSuccessHandler.java

```diff
@@ -124,7 +125,12 @@ public void onAuthenticationSuccess(HttpServletRequest httpServletRequest, HttpS
             SecurityContextUtils.switchCurrentUserOrganization(user, team, httpServletRequest, true);
             // If bastion is enabled, every user must hold a built-in project for create temporary SQL console
             if (bastionEnabled) {
-                projectService.createProjectIfNotExists(user);
+                ProjectEntity project = projectService
+                        .createProjectIfNotExists(user,
+                                "USER_PROJECT_" + user.getAccountName(),
```

Comment:
Is the original project name `'USER_PROJECT_xxx' + user.getAccountName()`?

Reply:
Yes, `USER_PROJECT_` was as a constant.